SuiteCareers/SuiteCareers/Models/Response.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SuiteCareers.Models
+{
+    public class Response
+    {
+        [Key]
+        public long responseId { get; set; }
+        public int response { get; set; }
+
+        public int questionId { get; set; }
```

Comment:
To stay with C# standards, ALL properties in model classes should use CamelCase with the first letter capitalized